clients/hadoopfs/src/main/java/io/lakefs/LakeFSFileSystem.java

```diff
@@ -140,9 +146,28 @@ public boolean mkdirs(Path path, FsPermission fsPermission) throws IOException {
 
     @Override
     public FileStatus getFileStatus(Path path) throws IOException {
-        LOG.debug("$$$$$$$$$$$$$$$$$$$$$$$$$$$$ getFileStatus, path: {} $$$$$$$$$$$$$$$$$$$$$$$$$$$$", path.toString());
-        FileStatus fStatus = new FileStatus(0, false, 1, 20, 1, path);
-        return fStatus;
+        Location loc = pathToLocation(path);
```

Comment:
What do you think of adding debug logging at the beginning in case we need to track the filesystem behavior in new environments? (in a different format than I used)